mx-semantics/main/calls/configuration.md

```diff
@@ -24,4 +24,11 @@ module MX-CALL-RESULT-CONFIGURATION
         <mx-call-result> .MxCallResult </mx-call-result>
 endmodule
 
+module MX-CALL-RETV-CONFIGURATION
```

Comment:
This might be a nitpick, but I'd suggest being descriptive with the module names. Reading the module, I understand that `RETV` is supposed to mean "Return Values", but sounds more like an acronym.

Reply:
Fixed